### DISCUSSION TITLE
Growl error from earlier versions of Openutau

### DISCUSSION THREAD

**Discussion by MiraiNushi** [stakira/OpenUtau#547] [Ideas]
When I was using Kasane Teto's Sakebi voicebank in Openutau, it would cause her to growl sometimes and I really liked it. I don't know how it happened but when I updated Openutau, it was gone, but I kinda miss it and I'm wondering if the developer could have an option to bring it back. Also another thing to note, when I use Defoko's VCV voicebank, it happens sometimes too. (Or maybe this was just a problem only I had.)

**Comment by YstLol** [comment]
well, use rikimi vb
